### PR TITLE
added watch later

### PR DESCRIPTION
**Pull request by anupkgurung**
added watch later button.
on adding video to watch later user can navigate to watch later tab.

watchlater

### CODE REVIEW

File: src/components/VideoCard/VideoCard.jsx [hunk 1]
```diff
@@ -1,20 +1,62 @@
...
+    const watchLater = async (id) => {
+        const encodedToken = localStorage.getItem("token");
+        const video = { id, title, videoIframe, thumbnail, creator, alt }
+        try{
+            const data = await axios.post("/api/user/watchlater",{video}
+            , { headers: { authorization: encodedToken } }
+            )
+            console.log(data)
+        }catch(error){
+            console.log({error})
```
Comment (ranitasahaa1230) [inline src/components/VideoCard/VideoCard.jsx]: Remove the `console.log` and  instead show the error on the UI

**Review by ranitasahaa1230 — APPROVED** [review]
Code looks clean
Functionality is working proper
Looks good to go!

**Review by surbhi84 — APPROVED** [review]
- Code is readable.
- Ui looks clean.
- Don't see any breaking changes, good to merge.

File: src/components/VideoCard/VideoCard.jsx (hunk 1 above)
Comment (dikshadhote) [inline src/components/VideoCard/VideoCard.jsx]: Api calls can be moved to other file, cam be imported here

**Review by dikshadhote — APPROVED** [review]
Code looks clean. Please deploy the code while raising PR. Its easy to check if things are working fine in deployed version.
You can deploy dev branch on netlify and other branches which you will create from dev, will be deployed with preview by netlify bot.

**Pull request merged**